docs/index.rst

```diff
@@ -159,7 +159,9 @@ The ``HTTPTokenAuth`` is a generic authentication handler that can be used with
 
     WWW-Authenticate: Token realm="Authentication Required"
 
-The ``verify_token`` callback receives the authentication credentials provided by the client on the ``Authorization`` header. This can be a simple token, or can contain multiple arguments, which the function will have to parse and extract from the string.
+The ``verify_token`` callback receives the authentication credentials provided by the client on the ``Authorization`` header. This can be a simple token, or can contain multiple arguments, which the function will have to parse and extract from the string. Example:
+
+`Authorization: Token <Token Value>` 
```

Comment:
This is misleading. Where does `Token` come from? The default is to use `Bearer` as authentication scheme. Maybe change it to:

`Authorization: <Scheme> <Token Value>`

And also note that the default scheme is `Bearer`.

Reply:
@miguelgrinberg all good if you wanna put that more generic example further up in the docs, I specified token here because this is under the section that specifies how to authenticate with tokens.

Reply:
Ah I see, Bearers are another token type... here it should be fine then?

Reply:
The word that you put before the token is the authentication scheme. It can be "Basic" or "Digest" for the two auth methods defined in the HTTP protocol (supported in the other authentication classes of this extension). The OAuth spec uses "Bearer" here. The Flask-HTTPAuth extension defaults to "Bearer" as well on the token auth, but you can change it to anything you want. At the end of the day this is just a word. It can be anything that is not "Basic" or "Digest".

Reply:
Ah ok so re the default of Bearer: if I don't provide an auth type, i.e.

`Authorization: <Some token value>`

Should it be expected to work? This did not seem to work for me when using token auth, I wasn't getting any error but I wasn't getting a token value passed into my auth callback.

It seems when [looking through the code](https://github.com/miguelgrinberg/Flask-HTTPAuth/blob/master/flask_httpauth.py#L65) that we always `split` the value of this header, which will blow up if we don't provide two fields (i.e. type and value) in the header value field. We rescue this error and do nothing if it blows up, which I think is why I was getting no error.

Reply:
> Should it be expected to work? 

Not with this package. If you don't specify a scheme then you need to use a different package to handle your authentication. This package follows the standard practice of including a scheme name and then the credentials, be it basic, digest or a token-based authentication scheme.